src/ec/packages/ec-component-accordion/ecl-accordion.html.twig

```diff
@@ -54,23 +54,35 @@
   {% if _items is not empty %}
     {% for _item in _items %}
       <h{{ _item.level|default(3) }} class="ecl-accordion__title">
-        {% include '@ecl-twig/ec-component-button/ecl-button.html.twig' with {
-          label: _item.toggle.label,
-          variant: 'ghost',
-          type: 'button',
-          icon: _item.toggle.icon,
-          icon_position: 'before',
-          extra_classes: 'ecl-accordion__toggle',
-          extra_attributes: [
-            { name: 'aria-controls', value: _item.id },
-            { name: 'data-ecl-accordion-toggle' },
-          ]
-        } only %}
+        <button
+          class="ecl-button ecl-button--ghost ecl-accordion__toggle"
+          type="button"
+          aria-controls="{{ _item.id }}-content"
+          data-ecl-accordion-toggle
+        >
+          <span class="ecl-button__container">
+            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
+              icon: {
+                path: _item.toggle.icon.path,
+                type: "general",
+                name: _item.toggle.icon.name,
+                size: "s"
+              },
+              extra_classes: 'ecl-icon ecl-icon--s ecl-button__icon ecl-button__icon--before',
+              extra_attributes: [{ name: "data-ecl-icon" }]
+            } only %}
+
+            <span class="ecl-button__label" data-ecl-label="true">
+              {{- _item.toggle.label -}}
+            </span>
+          </span>
+        </button>
+        
       </h{{ _item.level|default(3) }}>
       <div
         class="ecl-accordion__content"
         hidden
-        id="{{ _item.id }}"
+        id="{{ _item.id }}-content"
```

Comment:
Not really sure why this is done like this in ECL, might be that this a way to "target" the old accordion (this is the deprecated one).